src/Commands/DownloadCommand.php

```diff
@@ -16,7 +16,15 @@ class DownloadCommand extends Command
     public function handle() : int
     {
         $this->getLocales()->each(function ($locale, $key) {
-            $translations = app(Poeditor::class)->download(is_string($key) ? $key : $locale);
+            $translations = collect(app(Poeditor::class)->download(is_string($key) ? $key : $locale))
+                ->map(function ($translation) {
+                    if (is_string($translation)) {
+                        return $translation;
+                    }
+
+                    return collect($translation)->filter(fn ($value) => $value !== '');
+                })
+                ->toArray();
```

Comment:
I don't think this will work if you have a PHP translation file with more nesting, for example:
```php
// policies.php

return [
	'user' => [
		'view' => [
			'no_permission' => '',
		],
    ],
];
```
Can we maybe use the dot() collection method, followed by filter(), en finally undot() again? 🤔 (but what if translation key contains dots 🧐)

I would also move this to the download method of poeditor class (especially when we will also need this filtering in the "is everything up to date"-command)

Reply:
Ah yes good call. If we want to use the dot methods we're going to need to up the lowest laravel requirements though.

Reply:
In my PR I already dropped support for Laravel 9 because not only the absence of the dot-method caused issues